src/main/java/br/com/sbk/sbking/networking/server/Table.java

```diff
@@ -12,43 +12,50 @@
 import br.com.sbk.sbking.core.Player;
 import br.com.sbk.sbking.networking.server.gameServer.GameServer;
 
+/**
+ * This class has two responsibilities: 1: control the players and the
+ * directions they are sitting 2: host a gameServer. This way, the gameServer
+ * should remain isolated from the lower layers and only be altered via this
+ * table.
+ */
 public class Table {
 
-  private Map<Direction, ClientGameSocket> playerSockets;
-  private Collection<ClientGameSocket> spectatorSockets;
-  private Map<UUID, ClientGameSocket> allSockets;
+  private Map<Direction, Player> seatedPlayers;
+  private Collection<Player> spectatorPlayers;
+  private Map<UUID, Player> allPlayers;
   private GameServer gameServer;
 
   public Table(GameServer gameServer) {
     this.gameServer = gameServer;
-    this.playerSockets = new HashMap<Direction, ClientGameSocket>();
-    this.spectatorSockets = new ArrayList<ClientGameSocket>();
-    this.allSockets = new HashMap<UUID, ClientGameSocket>();
+    this.gameServer.setTable(this);
+    this.seatedPlayers = new HashMap<Direction, Player>();
+    this.spectatorPlayers = new ArrayList<Player>();
+    this.allPlayers = new HashMap<UUID, Player>();
   }
 
   public void moveToSeat(UUID from, Direction to) {
```

Comment:
I didn't like this variable names here. I can't interpretate UUID as `from`

Reply:
Also, I don't understand why this method exists, because we want to mode Players to Seats, not UUIDs. I saw that this method calls other

Reply:
The reasoning is the same as https://github.com/rulojuka/sbking/pull/88/files#r612096017
To the network, the "thing" that wants to go from a seat to other is an `UUID`. That is why `Table` has to deal with this.